lib-py/casmutils/xtal/structure.py

```diff
@@ -0,0 +1,194 @@
+from . import _xtal
+from .lattice import Lattice, MutableLattice
+from .site import Site, MutableSite
+
+class _Structure():
+
+    """Base class for both mutable and immutable
+    Structure classes. Defines function that are
+    common to both"""
+
+    def __init__(self, lattice, basis):
+        """creates an instance of _xtal.Structure
```

Comment:
The user doesn't know or care about `_xtal.Structure`. We're trying to hide these details.